.env.example

```
@@ -26,19 +33,19 @@ REDIS_PORT=6379
-WEBHOOK_CLIENT_STATE=your-secret-client-state-here
+WEBHOOK_CLIENT_STATE=your_webhook_client_state_here  # Run: node scripts/generate-secure-secrets.js webhook
 
 # Email Processing Configuration
 EMAIL_BATCH_SIZE=50
 SUBSCRIPTION_RENEWAL_MINUTES=30
 
 # JWT Configuration
-# SECURITY CRITICAL: Generate strong secret using: node scripts/generate-jwt-secret.js
-JWT_SECRET=your-jwt-secret-here-must-be-64-chars-minimum
+# SECURITY CRITICAL: Generate strong secret using: node scripts/generate-secure-secrets.js
+JWT_SECRET=your_jwt_secret_here  # Run: node scripts/generate-secure-secrets.js jwt
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)